standalone/source/main.cpp

```diff
@@ -33,7 +38,10 @@ int main(int argc, char** argv) {
   if (result["help"].as<bool>()) {
     std::cout << options.help() << std::endl;
     return 0;
-  } else if (result["version"].as<bool>()) {
+  }
+
+  // prevent warning: do not use 'else' after 'return' [readability-else-after-return]
+  if (result["version"].as<bool>()) {
```

Comment:
Tbh I don't see any readability improvement here.

Reply:
but it prevents this warning and the paragraph emphasize the previous return